### ISSUE TITLE
Ethical Problems with the use of this language

### ISSUE THREAD

**Issue by Petingoso** [ShibaDisk/Cring3#2]
The owner of this esoteric, wacky and dumb language, who is homosexual and butthurt at python programs taking 0.000001 nanoseconds longer than """real""" languages dares to insult Scratch, while knowing full well the only good language is **B**, not the retarded shit that is C.

Also he uses Windows 10

Fumo
![image](https://user-images.githubusercontent.com/92183955/143504804-21aaa60e-441c-4f0a-8b9a-3c70b0817cca.png)

**Comment by ShibaDisk** [comment]
> python programs taking 0.000001 nanoseconds longer than """real""" languages 

this is NOT my issue with python, my issue with python is that it does not have semicolons and its formatting is too strict, making the code less compact and makes it CONSOOM more space.
```
if(name == "Petingoso"):
     cringe = True
```
does not look as nice and compact as
```
if(name == "Petingoso"){ cringe = true; }
```
This is a fact. **REAL** and **BASED** languages use semicolons.
![le bread](https://user-images.githubusercontent.com/83471780/143505624-dcd21594-5c8b-4ea1-a522-3bbfca89319f.png)


In conclusion, there is no  ethical issue with this language, and you are stupid and dumb and cringe.
